### ISSUE TITLE
XForm with no relevant fields does not load

### ISSUE THREAD

**Issue by abbyad** [medic/medic#4249]
A form with a single page does not load if the style is set to `pages`. It gives the following error:
```
TypeError: Cannot read property 'scrollIntoView' of undefined
    at Object.focusOnFirstQuestion
```

The workaround is to remove the `pages` class (which is set by the Settings>style  in XLSForm).

## Example form
```
<?xml version="1.0"?>
<h:html xmlns="http://www.w3.org/2002/xforms" xmlns:ev="http://www.w3.org/2001/xml-events" xmlns:h="http://www.w3.org/1999/xhtml" xmlns:jr="http://openrosa.org/javarosa" xmlns:orx="http://openrosa.org/xforms" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
  <h:head>
    <h:title>Test Issue 4249: form with 1 page</h:title>
    <model>
      <instance>
        <issue_4249 delimiter="#" id="issue_4249" prefix="J1!issue_4249!" version="2018-02-28 10-31">
          <group>
            <field/>
          </group>
          <meta>
            <instanceID/>
          </meta>
        </issue_4249>
      </instance>
      <instance id="contact-summary"/>
      <bind nodeset="/issue_4249/group" relevant="./source = 'user'"/>
      <bind nodeset="/issue_4249/group/field" type="string"/>
      <bind calculate="concat('uuid:', uuid())" nodeset="/issue_4249/meta/instanceID" readonly="true()" type="string"/>
    </model>
  </h:head>
  <h:body class="pages">
    <group appearance="field-list" ref="/issue_4249/group">
      <label>Testing db:person object populating associated fields</label>
      <input ref="/issue_4249/group/field">
        <label>Test field</label>
      </input>
    </group>
  </h:body>
</h:html>
```

**Comment by abbyad** [comment]
Oops, the error is caused because there is no relevant question (because the relevant statement will never be true)!

**Comment by abbyad** [comment]
Renaming issue to state that form with no relevant fields has an error. This may not be worth fixing, but worth documenting and perhaps putting a useful error in console.

**Comment by garethbowen** [comment]
@abbyad Do you think this is a dupe of https://github.com/medic/medic/issues/5484 ?

**Comment by abbyad** [comment]
That's very likely! Both have forms without relevant content, and have the same error. We should test the form from this issue as part of AT for whatever solution is used for #5484.

**Comment by garethbowen** [comment]
I've tested this with the fix for #5484 and it now shows like this:

![Screenshot from 2019-09-10 13-21-40](https://user-images.githubusercontent.com/2064938/64576947-ed54eb80-d3ce-11e9-820a-aa11c511cea6.png)
![Screenshot from 2019-09-10 13-28-56](https://user-images.githubusercontent.com/2064938/64576956-f645bd00-d3ce-11e9-937f-5cacb0f901e8.png)

Closing as a duplicate.
